src/views/Schedule2.vue

```diff
@@ -88,4 +114,23 @@ export default Vue.extend({
     width: calc(100% - 20px);
   }
 }
+
+.now-bar {
+  position: absolute;
+  width: 100%;
+  height: 3px;
+  background-color: black;
+  z-index: 2;
+}
+
+.now-indicator {
+  display : inline-block;
+  height : 0;
+  width : 0;
+  border-top : 4px solid transparent;
+  border-bottom : 6px solid transparent;
+  border-left : 10px solid black;
+  position: absolute;
+  top: -3px;
+}
```

Comment:
We may remove `.now-indicator` to gain one DOM element; Could be replaced with:
```suggestion
.now-bar:before {
  content: "▶";
  position: absolute;
  top: -10px;
  left: -2px;
  color: $tertiary;
}

.now-bar:after {
  content: "◀";
  position: absolute;
  top: -13px;
  right: -4px;
  color: $tertiary;
}
```

Reply:
Fixed